milli/src/search/new/tests/stop_words.rs

```diff
@@ -259,7 +259,7 @@ fn test_ignore_stop_words() {
             ),
             Proximity(
                 Rank {
-                    rank: 7,
+                    rank: 1,
```

Comment:
Super surprised at this change. What happened?

Reply:
About this one, I was surprised by the old rank 🤔 the good rank should be `1` because there is only 1 searchable word in the query, no?

Reply:
let me check

Reply:
So there are 2 searchable words in the query, because the query is `to the dragon xyz`, the stop words are `["to", "The", "xyz"]` (note the capital T, stop words are case sensitive), so the possible matching words are "the" and "dragon".

The max_rank of proximity would be 1 if there was only a single word. Since the two matching words are very close ("the" and "dragon"), the expected result should indeed be close to the max (so 7 is more correct than 1)